titiler/xarray/factory.py

```diff
@@ -18,6 +20,78 @@
 from titiler.core.resources.responses import JSONResponse
 
 
+def xarray_open_dataset(
+    src_path: str,
+    z: Optional[str] = None,
```

Comment:
Z is an optional `String` but in the factory it's defined as an integer

Reply:
fixed